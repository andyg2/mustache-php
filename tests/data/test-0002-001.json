{"mustaches": [
	{"type": "Natural"},
	{"type": "Hungarian"},
	{"type": "Dali"},
	{"type": "English"},
	{"type": "Imperial"},
	{"type": "Freestyle", "last": "true"}
]}
