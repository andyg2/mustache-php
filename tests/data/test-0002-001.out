Natural, Hungarian, Dali, English, Imperial, and Freestyle
